src/displayapp/screens/WatchFaceTerminal.cpp

```diff
@@ -30,7 +30,6 @@ WatchFaceTerminal::WatchFaceTerminal(DisplayApp* app,
     settingsController {settingsController},
     heartRateController {heartRateController},
     motionController {motionController} {
-  settingsController.SetClockFace(3);
```

Comment:
could you make a small PR with just this cleanup line, for it to get merged faster

Reply:
I did it in #1423. Thank you! Should be a lot cleaner this way :)